source/framework/core/src/TRestEvent.cxx

```diff
@@ -73,26 +73,25 @@ void TRestEvent::Initialize() {
     fSubEventID = 0;
     fSubEventTag = "";
     fOk = true;
-
     fPad = nullptr;
 }
 
 //////////////////////////////////////////////////////////////////////////
 /// Set the time of the event
 ///
 void TRestEvent::SetTime(Double_t time) {
-    Int_t sec = (Int_t)time;
-    Int_t nsec = (Int_t)((time - sec) * 1E9);
+    auto seconds = (Int_t)time;
```

Comment:
Perhaps thiis is the issue? I don't know if `auto` is capable to assign `Int_t` but should be equivalent to `int`. Unless is doing something weird...